main.go

```diff
@@ -1,9 +1,8 @@
 package main
 
 import (
-	"fmt"
-
 	"github.com/crifff/test-ci/lib"
+	"github.com/k0kubun/pp"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/k0kubun/pp (can't find import: github.com/k0kubun/pp)

<!-- Comment by [SideCI](https://sideci.com) -->